x-pack/plugins/apm/common/latency_distribution_chart_types.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import * as t from 'io-ts';
+
+export enum LATENCY_DISTRIBUTION_CHART_TYPE {
+  TRANSACTION_DETAILS = 'TRANSACTION_DETAILS',
```

Comment:
in the client side, this chart type is used in `x-pack/plugins/apm/public/components/app/transaction_details/distribution/use_transaction_distribution_chart_data.ts`, it's the original latency distribution chart we had in transaction details before correlations came along. i could call it "transactionDistributionChartData" to match the name of the hook that references it. but "distribution" and "chart" are redundant since they are traits shared by all members of this enum. What would you suggest?

Reply:
Could also call it "trace samples" to be in line with the label in the selected tab

Reply:
What about 'transactionLatency'? At least we should avoid referencing it by where it is rendered rather than by what it renders.